docs/source/guide/tutorials/base/how_to_train/semantic_segmentation.rst

```diff
@@ -1,2 +1,238 @@
+                      --save-performance performance.json
+
+Finally, we get the validation output:
+
+.. code-block::
+
+  ...
+
+  2023-02-21 18:09:56,134 | INFO : run task done.
+  2023-02-21 18:09:57,807 | INFO : called evaluate()
+  2023-02-21 18:09:57,807 | INFO : Computing mDice
+  2023-02-21 18:09:58,508 | INFO : mDice after evaluation: 0.9659400544959128
+  Performance(score: 0.9659400544959128, dashboard: (1 metric groups))
+
+In ``performance.json`` file, the validation output score is saved as:
+
+.. code-block::
+
+  {"Dice Average": 0.9659400544959128}
+
+
+*********
+Export
+*********
+
+1. ``otx export`` exports a trained Pytorch `.pth` model to the OpenVINO™ Intermediate Representation (IR) format.
+It allows running the model on the Intel hardware much more efficient, especially on the CPU. Also, the resulting IR model is required to run POT optimization. IR model consists of 2 files: ``openvino.xml`` for weights and ``openvino.bin`` for architecture.
+
+2. We can run the below command line to export the trained model
+and save the exported model to the ``openvino_model`` folder.
+
+.. code-block::
+
+  (otx) ...$ otx export --load-weights models/weights.pth \
+                        --save-model-to openvino_model
+
+  ...
+
+  2023-02-02 03:23:03,057 | INFO : run task done.
+  2023-02-02 03:23:03,064 | INFO : Exporting completed
+
+
+3. We can check the mDice score of the IR model and the consistency between the exported model and the PyTorch model,
+using ``otx eval`` and passing the IR model path to the ``--load-weights`` parameter.
+
+.. code-block::
+
+  (otx) ...$ otx eval --test-data-roots splitted_dataset/val \
+                      --load-weights openvino_model/openvino.xml \
+                      --save-performance openvino_model/performance.json
+
+  ...
+
+  Performance(score: 0.9659400544959128, dashboard: (1 metric groups))
+
+
+*************
+Optimization
+*************
+
+1. We can further optimize the model with ``otx optimize``.
+It uses NNCF or POT depending on the model format.
+
+Please, refer to :doc:`optimization explanation <../../../explanation/additional_features/models_optimization>` section to get the intuition of what we use under the hood for optimization purposes.
+
+2. Command example for optimizing
+a PyTorch model (`.pth`) with OpenVINO™ NNCF.
+
+.. code-block::
+
+  (otx) ...$ otx optimize --load-weights models/weights.pth --save-model-to nncf_model
+
+  ...
+
+  INFO:nncf:Loaded 5286/5286 parameters
+  2023-02-21 18:09:56,134 | INFO : run task done.
+  2023-02-21 18:09:57,807 | INFO : called evaluate()
+  2023-02-21 18:09:57,807 | INFO : Computing mDice
+  2023-02-21 18:09:58,508 | INFO : mDice after evaluation: 0.9659400544959128
+  Performance(score: 0.9659400544959128, dashboard: (1 metric groups))
+
+The optimization time relies on the hardware characteristics, for example on 1 GeForce 3090 and Intel(R) Core(TM) i9-10980XE it took about 15 minutes.
```

Comment:
```suggestion
The optimization time relies on the hardware characteristics, for example on 1 NVIDIA GeForce RTX 3090 and Intel(R) Core(TM) i9-10980XE it took about 15 minutes.
```

Reply:
I would change but seems like align it with classification and detection.

Reply:
Classification and detection are wrong. Official name is `NVIDIA GeForce RTX 3090`.

Reply:
I didn't say this is wrong. I meant cls and det should be changed too..